fractal-puppy/Engine
# Tree Flower FX

# Feature
- Now trees in the Graveyard (from start to the bridge) have a particle system that simulates flowers falling from the trees. They have been splitted into 2 prefabs. One for small trees and one for bigger ones. This allows to apply changes way faster.
![FlowerTreeEffect](https://user-images.githubusercontent.com/25633904/62557379-2ac9d480-b877-11e9-8bf3-bffbbde6dc45.gif)

# Related Fixes
- Color over time and Size over time is correctly copied on duplicating gameobject.
- Prefab now work properly with Particles and Trails.

## Test
- Check that trees have that particle effect.
- Check that you can drag N drop one of those prefabs and that the particle effect is visualized correctly. Remember that due to exposure in Game window, the color of the texture is way darker in the scene window!
